app/src/main/res/layout/app_install_cardview_with_rating.xml

```diff
@@ -28,19 +28,36 @@
       android:id="@+id/card_info_install_layout"
       android:layout_width="match_parent"
       android:layout_height="match_parent"
-      android:padding="4dp"
+      android:padding="9dp"
       android:visibility="visible"
       >
 
     <TextView
-        android:id="@+id/app_title_textview"
+        android:id="@+id/app_title_textview_with_rating"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginEnd="8dp"
         android:layout_marginLeft="8dp"
         android:layout_marginRight="8dp"
         android:layout_marginStart="8dp"
         android:lines="1"
+        android:visibility="gone"
+        tools:text="@string/appc_title_settings_appcoins_wallet"
+        tools:visibility="visible"
+        style="@style/Aptoide.TextView.Medium.S.Black"
```

Comment:
Why do you have two textviews with the same content, just changing the title? Also, this textview is not being used.

Reply:
I had the intention to do a card with both titles and then change the visibility depending of it having rating or not, but then I gave up on that idea and forgot to delete that